samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties

```diff
@@ -178,6 +178,8 @@ samigo_formula_error_5=5\: Parte inesperada
 samigo_formula_error_6=6\: Final inesperado de la expresi\u00f3n 
 samigo_formula_error_7=7\: Valor esperado
 samigo_formula_error_9=9\: Variable no definida en las instrucciones
+samigo_formula_error_10=10: Variable no definida en el comentario a la respuesta correcta
```

Comment:
@pablosanroman can you help here whenever you can? Thanks

Reply:
@mpellicer All the translations are on the PR. This is ready to merge! :)

Reply:
@mpellicer 
+ samigo_formula_error_10=10:  Aldagaia ez dago definituta feedbackean (ongi dagoen erantzunarena)

Reply:
Thanks @PabloSanRoman, I've included a fix for this.